packages/web-comment/src/utils/formatAddress.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	formatAddress: (appeal) => {
+		const addressComponents = [
+			appeal.siteAddressLine1,
+			appeal.siteAddressLine2,
+			appeal.siteAddressTown,
+			appeal.siteAddressPostcode
+		];
+
+		return addressComponents.filter((component) => !!component).join(', ');
```

Comment:
I prefer `filter(Boolean)` but either is fine!

Reply:
hadn't seen this before! Changed it to this.